uiowa/uiowa
# i3056 Style adjustment for APR pages

This PR resolves https://github.com/uiowa/uiowa/issues/3056.

# How to test

spin up `pharmacy.uiowa.edu`
hit this page: https://pharmacy.local.drupal.uiowa.edu/people/jay-d-currie
IF YOU ARE HAVING PROBLEMS, YOU NEED TO MAKE SURE THE ABOVE PAGE IS HTTPS
Verify that the headings are smaller than https://pharmacy.prod.drupal.uiowa.edu/people/jay-d-currie, and are similar to https://tippie.uiowa.edu/people/pamela-g-bourjaily
